test/e2e/kind_e2e_test.go

```diff
@@ -60,6 +60,38 @@ func TestKindSuite(t *testing.T) {
 	e2e.Run[kindEnv](t, &kindSuite{}, e2eParams...)
 }
 
+const (
```

Comment:
Can we put this const in [common_test.go](https://github.com/DataDog/datadog-operator/blob/48f959c0728546560feb7ee324f4b17afe8ec24d/test/e2e/common_test.go)?